src/Plugin.php

```diff
@@ -65,6 +65,9 @@ public static function init() {
     // Add teaser image to RSS feeds.
     add_action('rss2_item', __NAMESPACE__ . '\Feed::rss2_item');
 
+    // Fix Facebook embeds to keep within wrapper borders.
+    add_filter('oembed_fetch_url', __CLASS__ . '::oembed_fetch_url', 10, 3);
```

Comment:
About 1, it's true but makes no sense that this is before the `is_admin()` exit point. Maybe we can move some of this filters after the `is_admin()` return? Otherwise, we can move this up, but we will be adding the filter in admin section where not necessary.